cmd/ttn-lw-stack/commands/start.go

```diff
@@ -269,6 +270,18 @@ var startCommand = &cobra.Command{
 					Redis: redis.New(config.Redis.WithNamespace("as", "io", "webhooks")),
 				}
 			}
+			switch cfg := config.AS.DeviceCache; cfg.Backend {
+			case "memory":
+				var cache gcache.Cache
+				if cfg.Size <= 0 {
+					cache = gcache.New(-1).Build()
```

Comment:
Consider building the cache itself after setting these options (for example, due to duplication, this one doesn't have `LFU()`.

Reply:
If the cache size is not bound, then it will never evict entries, so no `LFU()` is needed. In fact, gcache will refuse to create the cache if `LFU()` is set.